mail-client/neomutt-test-files/neomutt-test-files-1.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+UPSTREAM_COMMIT="8629adab700a75c54e8e28bf05ad092503a98f75"
+SRC_URI="https://github.com/neomutt/${PN}/archive/${UPSTREAM_COMMIT}.tar.gz -> ${P}.tar.gz"
+
+DESCRIPTION="Test files for the main NeoMutt source"
+HOMEPAGE="https://neomutt.org/"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+S="${WORKDIR}/${PN}-${UPSTREAM_COMMIT}"
+
+src_unpack() {
+	if [[ -n ${A} ]]; then
+		mkdir --verbose -p "${S}" || die "Could not create ${S}"
+	fi
+}
+
+src_prepare() {
+	if [[ -n ${A} ]]; then
+		cd "${DISTDIR}" || die "Could not cd into DISTDIR"
+		cp --verbose ${A[@]} "${S}" || die "Could not copy package"
+	fi
+	eapply_user
+}
```

Comment:
All of this seems rather redundant to me. It should just work if you define `S`, as you've done?

Rest look ok!

Reply:
Maybe. I don't care about unpacking the source though. I would like to package the tar file so that I can unpack it during the testing phase of the neomutt package. Defining `S` would not help there. Or am I missing something?

Reply:
Okay, would it then be possible to just do this in the neomutt ebuild? If I understood correctly, the contents of this package is just required on neomutt for testing phase, and not really needed to be installed on / to run the tests?

Reply:
Well, that's a good point :smile: I hadn't thought of that. Please have a look at the updated PR.